pkg/execute/executor.go

```diff
@@ -188,9 +181,20 @@ func (e *DefaultExecutor) Execute() interactive.Message {
 		return empty
 	}
 
+	if e.kubectlCmdBuilder.CanHandle(args) {
```

Comment:
Couldn't test it fully now, but I think in current implementation of command reporting (analytics), we can't distinguish `kubectl get services/pods/deployments` clicked via Help vs `kubectl get services/pods/deployments` executed with the interactive kubectl.
I think it's time to introduce the enum now - or anything what makes us know that the origin is "interactive kubectl". For me, it's a really nice experimental feature, so we'll know if it is used or not.

Reply:
tbh I don't see an option to do that. Let me know how you would like to implement that. For now we will have analytics that someone was using the command builder as we will report the `kc-cmd-builder`. I will also try to think about some option to distinguish those buttons clicks.

Reply:
> For now we will have analytics that someone was using the command builder as we will report the kc-cmd-builder.

Oh, I didn't see that in the Segment.io Debugger console, but you know how the console works, so... If this is reported as well, then it's fine for me 👍

Reply:
I also updated the command that is sent from `kc-cmd-builder` to:
- "kc-cmd-builder --verbs"
- "kc-cmd-builder --resource-type"
- "kc-cmd-builder --resource-name"
- "kc-cmd-builder --namespace"

(depends on which dropdown was changed)

Reply:
Can we easily strip out actual resource name and namespace?
![image](https://user-images.githubusercontent.com/7155799/195312407-6902c517-0c60-4533-8114-1ea93cc55d1f.png)
![image](https://user-images.githubusercontent.com/7155799/195312938-5eaa1316-ed96-40f0-a3fe-23483db1dee4.png)

If not, let's go back to the previous impl, as this one would leak potentially private data 👍

Reply:
Yes, we can 👍  I will fix that too 👍